package.json

```diff
@@ -36,7 +36,8 @@
         "lint": "eslint -c ./.eslintrc --fix --ext .jsx,.js,.vue ./components",
         "lint:style": "stylelint \"./examples/**/*.less\" --fix --syntax less",
         "commitmsg": "commitlint -x @commitlint/config-conventional -e $GIT_PARAMS",
-        "codecov": "codecov"
+        "codecov": "codecov",
+        "postinstall": "opencollective-postinstall"
```

Comment:
`opencollective-postinstall: not found`

Is `opencollective-postinstall` missing?